mrp_project/wizard/mrp_production_createproject.py

```diff
@@ -0,0 +1,74 @@
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class MrpProductionCreateProject(models.TransientModel):
+    """ wizard to create a Project from a Manufacturing Order """
+    _name = "mrp.production.createproject"
+
+    @api.model
+    def default_get(self, fields):
+        result = super(MrpProductionCreateProject, self).default_get(fields)
+        mrp_production_id = self.env.context.get('active_id')
+        if mrp_production_id:
+            result['mrp_production_id'] = mrp_production_id
+        return result
+
+    mrp_production_id = fields.Many2one(
+        'mrp.production',
+        string='Production',
+        domain=[('type', '=', 'production')]
+    )
+    project_id = fields.Many2one(
+        'project.project',
+        string='Project',
```

Comment:
Should bbe remove the string because the system automatically compute the name of field, so we don't need define it again!